<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>series_editor</class>
 <widget class="QWidget" name="series_editor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>358</width>
    <height>215</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Series Editor</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="8" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="scan_range_label">
       <property name="text">
        <string>Scan range:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="scan_range_start">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>1000000</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="scan_range_stop">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>1000000</number>
       </property>
       <property name="value">
        <number>3</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="1">
    <widget class="QSpinBox" name="skip_frames">
     <property name="maximum">
      <number>10000</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="scan_shape_label">
       <property name="text">
        <string>Scan shape:</string>
       </property>
       <property name="buddy">
        <cstring>scan_shape_i</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="scan_shape_i">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>10000</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="scan_shape_j">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>10000</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="series_dir_label">
       <property name="text">
        <string>Directory:</string>
       </property>
       <property name="buddy">
        <cstring>series_dir</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="series_dir"/>
     </item>
     <item>
      <widget class="QPushButton" name="select_series_dir">
       <property name="text">
        <string>Select</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="skip_frames_label">
     <property name="text">
      <string>Skip frames:</string>
     </property>
     <property name="buddy">
      <cstring>skip_frames</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="scan_shift">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Shift scan indices to account for any misalignment in this series.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>Scan shift:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="scan_shift_i">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Shift scan indices to account for any misalignment in this series.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="minimum">
        <number>-100000</number>
       </property>
       <property name="maximum">
        <number>100000</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="scan_shift_j">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Shift scan indices to account for any misalignment in this series.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="minimum">
        <number>-100000</number>
       </property>
       <property name="maximum">
        <number>100000</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>series_dir</tabstop>
  <tabstop>select_series_dir</tabstop>
  <tabstop>scan_shape_i</tabstop>
  <tabstop>scan_shape_j</tabstop>
  <tabstop>scan_range_start</tabstop>
  <tabstop>scan_range_stop</tabstop>
  <tabstop>scan_shift_i</tabstop>
  <tabstop>scan_shift_j</tabstop>
  <tabstop>skip_frames</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
